package.json

```diff
@@ -2,9 +2,13 @@
   "dependencies": {
     "minimatch": "^3.0.5",
     "moment": "^2.29.4",
+    "playwright": "^1.35.1",
     "scroll-to-bottomjs": "^1.1.0",
-    "selenium-webdriver": "^3.6.0"
+    "@lambdatest/selenium-driver": "^1.0.0",
+    "@lambdatest/smartui-cli": "^2.0.0",
+    "selenium-webdriver": "^4.16.0"
   },
+  "type": "module",
```

Comment:
Is there any specific need, to make this complete project as module?

Reply:
@pinanks has specifically asked to add the same.